backend/schema.js

```diff
@@ -5,13 +5,15 @@ export default gql`
     id: ID!
     title: String!
     author: String!
+    pages: String
```

Comment:
```suggestion
    numPages: Number
```
This is primarily a name change; if you accept it would of course require additional changes downstream where we use this field.

Reply:
I change the attribute's name to numPages and it's type to **Int**.